guides/v2.0/contributor-guide/contributing.md

```diff
@@ -37,9 +37,9 @@ This contribution model has contributors maintaining their own copy of the forke
 Contributions can take the form of new components or features, changes to existing features, tests, documentation (such as developer guides, user guides, examples, or specifications), bug fixes, optimizations, or just good suggestions.
 
 
-The Magento 2 development team reviews all issues and contributions submitted by the community of developers in a first-in, first-out basis. During the review we might require clarifications from the contributor. If there is no response from the contributor for two weeks, the issue is closed.
```

Comment:
change "in" to "on"

Reply:
the text in red is deleted. What should I change?